neps/nep-0001.md

```diff
@@ -346,31 +388,55 @@ pull request. Fork the `NEP repository`, make your ownership modification,
 and submit a pull request. You should mention both the original author and
 ``@near/nep-editors`` in a comment on the pull request. 
 
-## NEP Editors
+## Roles, Ownership, and Accountability
+The overall NEP process should balance democracy and inclusivity, with accountability. This means the proposal needs to have owners – people who carry the burden of proving “beyond reasonable doubt” that the proposed design does what it claims to do and that it does not negatively affect our technological values or impede future velocity. The owners of the proposal are the author and its potential proponents. It is their responsibility to prove that the proposal works, and it is not responsibility of others to prove that it does not work. The default decision for any protocol change should be a No.
+
+There are defined administators who will decide when the proposal is accepted or rejected. In the long run, this will be a council elected by the stakeholders, which most likely will be only the validators and people that validators consider as an authority or experts, since validators have an ultimate control of deciding which protocol version to run on their machines. In the short term, process admins will be a preselected group of people. Admins will also be moderating the discussion once the proposal is submitted to make sure the discussion is civil and focused.
```

Comment:
>Admins will also be moderating the discussion once the proposal is submitted to make sure the discussion is civil and focused

If we have extra people power, it would be much better to have a dedicated moderation team. Admins themselves should be moderated, and that shouldn't involve conflict of interests.

To bootstrap the process, I think it's OK to lump the two responsibilities together. 

But maybe we can say something like "there's admin team, who does this" and "there's mod team, who does that", and than, separately "to bootstrap the process, both teams would intersect".